src/rpc/util.cpp

```diff
@@ -260,7 +260,7 @@ class DescribeAddressVisitor : public boost::static_visitor<UniValue>
         UniValue obj(UniValue::VOBJ);
         obj.pushKV("iswitness", true);
         obj.pushKV("witness_version", (int)id.version);
-        obj.pushKV("witness_program", HexStr(id.program, id.program + id.length));
+        obj.pushKV("witness_program", HexStr(Span<const unsigned char>(id.program, id.length)));
```

Comment:
Maybe we want to give `WitnessUnknown` `.data()` and `.size()` fields to avoid having to do this explicitly. Don't know. Probably not in thie PR though.